File: file_system/file_system_task.go
package main

import (
    "os"
    "io/ioutil"
    "strconv"
)

func main() {
    binaryData, err := ioutil.ReadFile("counter.txt")
    if err != nil {
	file, _ := os.Create("counter.txt")
	os.Chmod("counter.txt", 0777)
	file.Write([]byte {'1'})
	file.Close()
    } else {
	data := string(binaryData)
	num, err := strconv.Atoi(data)
	if err != nil {
	    os.Exit(1)
	} else {
	    num++
	    file, _ := os.OpenFile("counter.txt", os.O_RDWR, 0777)
	    file.WriteAt([]byte(strconv.Itoa(num)), 0)
	    file.Close()
	}
    }
}

File: file_system/file_system_task.py
import sys, os

def main():
    invalid_data = False
    try:
        file = open("counter.txt")
        num = file.read()
        if num.isdigit():
            num = int(num) + 1
            file = open("counter.txt", "w")
            file.write(str(num))
            file.close()
        else:
            invalid_data = True
    except:
        if not invalid_data:
            file = open("counter.txt", "w")
            file.write("1")
            file.close()
            os.chmod("counter.txt", 0o777)
    if invalid_data:
        sys.exit(1)

if __name__=="__main__":
    main()